app/views/contemporarypoets/browse_facet.html.erb

```diff
@@ -4,6 +4,8 @@
 
 <h1><%= t "browse.browse_type", default: "Browse by" %> <%= @browse_facet_info["label"] %></h1>
```

Comment:
Is it still necessary to overwrite this file? the behavior seems like it's back to the default